src/bindings/python/setup.cfg

```
@@ -17,7 +17,7 @@ passenv =
 commands=
   {envbindir}/python setup.py bdist_wheel
   {envbindir}/pip install --no-index --pre --find-links=dist/ openvino
-  pytest --backend={env:OV_BACKEND} tests -v -k 'not _cuda' --ignore=tests/test_onnx/test_zoo_models.py --ignore=tests/test_utils --ignore=tests/test_inference_engine
+  pytest --backend={env:OV_BACKEND} tests -v -k 'not _cuda' --ignore=tests/test_onnx/test_zoo_models.py --ignore=tests/test_utils --ignore=tests/test_runtime
```

```suggestion
  pytest --backend={env:OV_BACKEND} tests -v -k 'not _cuda' --ignore=tests/test_onnx/test_zoo_models.py --ignore=tests/test_utils
```
What happens if there is no ignore? Let's check it.
CC @p-wysocki